tests/posix/xsi_realtime/prj.conf

```
@@ -0,0 +1,19 @@
+CONFIG_POSIX_API=y
+CONFIG_ZTEST=y
+CONFIG_LOG=y
+
+CONFIG_POSIX_AEP_CHOICE_BASE=y
+CONFIG_POSIX_MESSAGE_PASSING=y
+CONFIG_POSIX_SHARED_MEMORY_OBJECTS=y
+CONFIG_POSIX_SYNCHRONIZED_IO=y
```

Should be able to replace these soon.
```suggestion
CONFIG_XSI_REALTIME=y
```